tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1406,3 +1417,66 @@ def test_verify_attribute_of_users_are_updated(session, ldap_data, ldap_tear_dow
         assert ldap_data['ldap_user_name'] in user_values['user']['firstname']
         assert ldap_data['ldap_user_name'] in user_values['user']['lastname']
         assert ldap_data['ldap_user_name'] in user_values['user']['mail']
+
+
+@pytest.mark.skip_if_open("BZ:1812688")
+@tier2
+def test_userlist_with_external_admin(session, auth_source_ipa, ldap_tear_down, groups_teardown):
+    """All the users should be displayed (internal and external) to all LDAP admin's either
+        it could be external (remote) or internal (Local)
+
+    :id: 7c7bf34a-06f9-11eb-b174-d46d6dd3b5b2
+
+    :steps:
+        1. Create 2 groups on an IPA server: e.g. sat_admins and sat_users.
+        2. On IPA, create user e.g. "idm_admin" and add to sat_admins.
+        3. On IPA, create user e.g. "idm_user" and add to sat_users.
+        4. Create IPA/IDM auth source
+        5. Create 2 local groups on Satellite: sat_admins and sat_users, link them to their
+            counterparts from IPA.
+        6. Assign all roles containing "view" or "read" to the sat_users group
```

Comment:
Is step 6 correct? It doesn't describe what is actually happening in test.

Reply:
I will update that, basically, we wanted to have a user with non-admin permissions